File: Zeze/Arch/LoginArgument.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Zeze.Arch
{
    public class LoginArgument : EventArgs
    {
        public string Account { get; set; }
        public string ClientId { get; set; }
    }
}

File: ZezeJava/ZezeJava/src/main/java/Zeze/Arch/LoginArgument.java
package Zeze.Arch;

import Zeze.Util.EventDispatcher;
import org.jetbrains.annotations.NotNull;

public class LoginArgument implements EventDispatcher.EventArgument {
	public final @NotNull String account;
	public final @NotNull String clientId;

	public LoginArgument(@NotNull String account, @NotNull String clientId) {
		this.account = account;
		this.clientId = clientId;
	}
}
